digitaltwins/device/src/digital_twin_client.ts

```diff
@@ -231,6 +232,10 @@ export class DigitalTwinClient {
     }, actualCallback);
   }
 
+  getVersion(): string {
+    return packageJson.version
+  }
```

Comment:
Does "c" or Python expose such an API?

Reply:
C does have a function GetVersionString();

Reply:
You are very correct.  My bad

---
In reply to: [419691523](https://github.com/Azure/azure-iot-sdk-node/pull/779#discussion_r419691523) [](ancestors = 419691523)